README.md

```diff
@@ -1,149 +1,62 @@
 
-Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+The project can be built with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli). Please install Arm Mbed CLI.
 
-## Import the example application
-
-From the command-line, import the example:
-
-```
-mbed import mbed-os-example-blinky
-cd mbed-os-example-blinky
-```
-
-### Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the Arm Compiler:
-
-```
-mbed compile -m K64F -t ARM
-```
-
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+Depending on the target, the example project can be built with GCC_ARM, ARM or IAR toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain support your target.
 
+```bash
+$ mbed compile -S
 ```
-[snip]
 
-Image: ./BUILD/K64F/GCC_ARM/mbed-os-example-blinky.bin
-```
-
-### Program your board
-
-1. Connect your Mbed device to the computer over USB.
-1. Copy the binary file to the Mbed device.
-1. Press the reset button to start the program.
-
-The LED on your platform turns on and off. The main thread will additionally take a snapshot of the device's runtime statistics and display it over serial to your PC. The snapshot includes:
-
-* System Information:
-    * Mbed OS Version: Will currently default to 999999
-    * Compiler ID
-        * ARM = 1
-        * GCC_ARM = 2
-        * IAR = 3
-    * [CPUID Register Information](#cpuid-register-information)
-    * [Compiler Version](#compiler-version)
-* CPU Statistics
-    * Percentage of runtime that the device has spent awake versus in sleep
-* Heap Statistics
-    * Current heap size
-    * Max heap size which refers to the largest the heap has grown to
-* Thread Statistics
-    * Provides information on all running threads in the OS including
-        * Thread ID
-        * Thread Name
-        * Thread State
-        * Thread Priority
-        * Thread Stack Size
-        * Thread Stack Space
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-#### Compiler Version
+Modifying default Mbed OS configuration parameters can be done at application level using an application configuration file. By default ARM Mbed CLI looks for [`mbed_app.json`](https://os.mbed.com/docs/mbed-os/latest/reference/configuration.html), however the configuration file can be named anything. It can be passed to ARM Mbed CLI using the optional argument `--app-config` of the `compile` sub-command. This project comes with configuration files for the different variants and configurations of the blinky application.
```

Comment:
There is only one variant left so rephrase the last sentence to "This project comes with a configuration file for one variant of the blinky application."

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/2f96047ab8eb914c0ce2ee6805a598a1df3445e4).